### PR TITLE
Add latest features from CloudFormation

### PR DESCRIPTION
**Pull request by michaelwittig**
* CloudWatch Alarm Auto Scaling Filter (https://github.com/marbot-io/monitoring-jump-start/pull/21)
* Set Security Hub Findings to notified (https://github.com/marbot-io/monitoring-jump-start/pull/33)
* Add Savings Plans monitoring (https://github.com/marbot-io/monitoring-jump-start/pull/31)

### CODE REVIEW

File: lambda/cloud-watch.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const AWS = require('aws-sdk')
+const cloudwatch = new AWS.CloudWatch({apiVersion: '2010-08-01'});
+const sns = new AWS.SNS({apiVersion: '2010-03-31'});
+const STATE2ACTION = {
+  'ALARM': 'AlarmActions',
+  'OK': 'OKActions',
+  'INSUFFICIENT_DATA':  'InsufficientDataActions'
+};
+exports.handler = async (event) => {
+  console.log(JSON.stringify(event));
```
Comment (andreaswittig) [inline lambda/cloud-watch.js:11]: Looks like debug logs? Remove or add log message?

File: lambda/cloud-watch.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const data = await cloudwatch.describeAlarms({
+    AlarmNames: [event.detail.alarmName],
+    MaxRecords: 1
+  }).promise();
+  const alarms = [...data.CompositeAlarms, ...data.MetricAlarms];
+  console.log(JSON.stringify(alarms));
```
Comment (andreaswittig) [inline lambda/cloud-watch.js]: Looks like debug logs? Remove or add log message?

File: main.tf [hunk 3]
```diff
@@ -1106,6 +1333,131 @@ resource "aws_cloudwatch_event_target" "macie_alert" {
...
+  }
+  EOF
+}
+
+resource "aws_lambda_function" "security_hub_workflow" {
+  count = (var.security_hub_finding && var.enabled) ? 1 : 0
+
+  filename         = data.archive_file.lambda.output_path
+  source_code_hash = data.archive_file.lambda.output_base64sha256
+  function_name    = "marbot-basic-security-hub-finding-workflow-${random_id.id8.hex}"
```
Comment (andreaswittig) [inline main.tf:1392]: Does `random_id` create a random ID only once? Or does this re-create the resource on every `terraform apply`? Could not find any hints in the documentation. Did you test that?

**Review by andreaswittig — APPROVED** [review]

File: main.tf (hunk 3 above)
Comment (michaelwittig) [reply main.tf:1392]: the random id stays stable once generated.

**Pull request merged**